front/src/modules/trainschedule/helpers/pacedTrain.ts

```diff
@@ -49,3 +49,55 @@ export const getExceptionChangeGroups = (
   Object.keys(pacedTrainExceptions).map(
     (exception) => exceptionChangeGroupsDict[exception as keyof OccurrenceException]
   );
+
+export const formatPacedTrainWithOccurenceDetails = (
+  pacedTrain: PacedTrainWithDetails,
+  exceptions: PacedTrainException
```

Comment:
Quite confusing to have PacedTrainException, I though at first it was the array of exceptions of the paced train. Can we use OccurrenceException here ?
```suggestion
  exceptions: OccurrenceException
```

Reply:
Well the type comes from editoast :/

Reply:
Done